README.md

```diff
@@ -1,127 +1,50 @@
-
-For more information on how to use the new APIs exposed in Floodlight v1.2, please refer to the OpenFlowJ-Loxi documentation and examples here (https://floodlight.atlassian.net/wiki/display/floodlightcontroller/How+to+use+OpenFlowJ-Loxigen).
-
-There are many more minor details, which can be found in the release notes. I have been grateful to have the support of many Floodlight developers, and together we have worked to provide the highest quality release within a reasonable time frame. I would especially like to thank the following individuals and beta testers for their code contributions and debugging efforts:
-
-https://floodlight.atlassian.net/wiki/display/floodlightcontroller/Authors+and+Contributors
+Floodlight is the leading open source OpenFlow controller. It is [supported by a community of developers](https://floodlight.atlassian.net/wiki/display/floodlightcontroller/Authors+and+Contributors), including a number of engineers from [Big Switch Networks](http://www.bigswitch.com/).
 
-Based on further community feedback, there will be minor releases to address any issues found or enhancements anyone would like to contribute. The mailing list has seen quite an uptick in activity over the last few months! 
+What is OpenFlow?
+-----------------
 
-If at any time you have a question or concern, please reach out to us. We rely on our fellow developers to make the most effective improvements and find any bugs. Thank you all for the support and I hope you find your work with Floodlight v1.2 fun and productive! 
+OpenFlow is a open standard managed by Open Networking Foundation. It specifies a protocol by which a remote controller can modify the behavior of networking devices through a well-defined “forwarding instruction set”. Floodlight is designed to work with the growing number of switches, routers, virtual switches, and access points that support the OpenFlow standard.
 
-Happy coding!  
-Ryan Izard  
-ryan.izard@bigswitch.com  
-rizard@g.clemson.edu
+Documentation and Support
+-------------------------
 
+Ready to get started using Floodlight? The [Floodlight wiki](https://floodlight.atlassian.net/wiki/spaces/floodlightcontroller/overview) contains user and developer documentation, as well as helpful tutorials from beginner to advanced.
 
+Do you have a question, comment, or a great idea you'd like to propose to the community? Please subscribe and send to our mailing list [floodlight@groups.io](mailto:floodlight@groups.io). Archives and additional content can be found on the [group homepage](https://groups.io/g/floodlight).
 
+Releases and Source Code
```

Comment:
I feel like we should just remove this section. We can rely on Github's release mechanism for documenting different releases if people want to use an older release. Otherwise we would prefer they use master

Reply:
Sure, we can do that